pkg/client/config.yml

```diff
@@ -39,6 +39,30 @@ system:
       list: '.mcpServers | to_entries | map(.value + {"name": .key})'
       set: .mcpServers[$NAME] = $JSON
       del: del(.mcpServers[$NAME])
+  cline:
+    displayName: Cline
+    source: https://github.com/cline/cline
```

Comment:
This is probably fine but we show this url in the UI when we can not detect a cline install.  So if you want to use a url that jumps straight to install instructions, that could also be a good choice.